code/game/machinery/doors/windowdoor.dm

```diff
@@ -24,6 +24,9 @@
 	..()
 	if(set_dir)
```

Comment:
Because it uses meaningful extra arguments (`set_dir` in this case).
I decided not to touch any of the /New procs like this to limit the scope, and leave those for another pass in the future.

Reply:
Initialize also allows for passing parameters just like `New`. Should be fine to add it after `mapload`

Reply:
Yeah, i know! Again, it was more of a scope-limiting measure, so that i don't try to refactor ALLL the Initialize uses, get bored midway, give up and never PR even the partial improvement😛